lib/psulib_blacklight/redis_config.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module PsulibBlacklight
+  class RedisConfig
+    def host
+      ENV.fetch('REDIS_HOST', 'localhost')
+    end
+
+    def port
+      ENV.fetch('REDIS_PORT', 6379)
+    end
+
+    def database
+      ENV.fetch('REDIS_DATABASE', 0)
```

Comment:
I think we should pull values out of `Settings` here since we are using the config gem. this will allow configuration to be put in the environment, or via config gem, this will allow deployment on linux and containers. maybe something like Settings.redis.host || ENV.fetch?